client-js/src/client/http-endpoint.js

```diff
@@ -231,92 +181,153 @@ export class HttpEndpoint extends Endpoint {
    * Sends off a command to the endpoint.
    *
    * @param {!TypedMessage<Command>} command a Command send to Spine server
-   * @return {Promise<Object>} a promise of a successful server response JSON data, rejected if
-   *                           the client response is not 2xx
+   * @return {Promise<JSON|SpineWebError>} a promise of a successful server response JSON data, rejected if
```

Comment:
JSON is not a valid type. Its a class with `stringify` and `parse` methods.
This should be an `Object`.